frontend/src/router/index.ts

```diff
@@ -79,6 +80,15 @@ const routes: RouteRecordRaw[] = [
           return auth.isAdmin(); //TODO: reroute to NotFound Screen/ AccessDeniedScreen?
         }
       },
+      {
+        path: 'auditlog',
+        component: AuditLog,
+        props: (route) => ({ token: route.query.token }),
+        beforeEnter: async () => {
+          const auth = await authPromise;
+          return auth.isAdmin(); //TODO: reroute to NotFound Screen/ AccessDeniedScreen?
+        }
```

Comment:
instead of repeating this on every admin resource, shouldn't we rather add some nesting? Using paths such as `/admin/auditlog` and `/admin/settings`, with the latter being the default when visiting `/admin/`.